src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -41,6 +43,8 @@
     <file src="Microsoft.DotNet.Build.Tasks\NativeBinaries\amd64\git2-69db893.dll" target="lib\NativeBinaries\amd64" />
     <file src="Microsoft.DotNet.Build.Tasks\NativeBinaries\x86\git2-69db893.dll" target="lib\NativeBinaries\x86" />
     <file src="Microsoft.DotNet.Build.Tasks\AssemblyInfoPartial.*" target="lib" />
+    <file src="GenFacades\GenFacades.dll" target="lib" />
```

Comment:
I do plan to clean-up the GenFacades projects and also delete their packages. However I'm going to do that in a separate PR.